### ISSUE TITLE
environments created using conda get recognized in anaconda-navigator but fail to activate

### ISSUE THREAD

**Issue by aerdengiz** [conda/conda#9779]
Hello,
I'm using anaconda-navigator 1.9.12 with Ubuntu 18.04LTS
I had to use conda to create a virtual environment for opencv as the navigator itself would figure out the right packages to retreive. Once this virtual environment was created using conda from the command line, it is visible from anaconda-navigator:
![image](https://user-images.githubusercontent.com/48405279/77335830-fa13ae00-6d26-11ea-9d05-d86232e9bf9a.png)
If I switch to the opencv environment from anaconda-navigator, it behaves as it does the switch but when I run a notebook, it reports that the packages (e.g. opencv) do not exist. In other words, even though the navigator thinks it has switched environments, nothing has budged.

However, if I try to activate this environment from conda and launch anaconda navigator, anaconda navigator fails to start. Instead I have to launch directly jupyter notebooks from the command line in which case it does indeed go to the correct environment.
![image](https://user-images.githubusercontent.com/48405279/77336425-cb4a0780-6d27-11ea-87c8-42e20bd6abca.png)
Basically, I either have to use conda but other than the (base) virtual environment I cannot use anaconda navigator, or I have to use anaconda-navigator, but can't switch to a different virtual environment through the navigator.

Note that the same problem exists with a brand new virtual environment that I created named akida...
Thank you for your help
A. Erdengiz
